e2e/tests/transfer/base_test.go

```diff
@@ -208,6 +222,56 @@ func (s *TransferTestSuite) TestMsgTransfer_Timeout_Nonincentivized() {
 	})
 }
 
+// TestSendEnabledParam tests changing ics20 SendEnabled parameter
+func (s *TransferTestSuite) TestSendEnabledParam() {
```

Comment:
Should I merge my prs first? Otherwise you can remove these changes, and I can readjust

Reply:
You can go ahead and merge your PRs I think, I can update this PR afterwards (which will just include the moving)